### DISCUSSION TITLE
Should results of typecheck be cached? `npm run dev` too slow?

### DISCUSSION THREAD

**Discussion by liammulh** [numberscope/frontscope#151] [Ideas]
This doesn't really rise to the level of an issue, but it might make sense to cache the results of `npm run typecheck` (as of this writing `vue-tsc --noEmit -p tsconfig.vitest.json --composite false`) so that `npm run dev` is faster. This would make development a bit more convenient.

**Comment by gwhitney** [comment]
I feel like after the recent spate of dependency updates, something like this must have become the default, because `npm run dev` does seem much faster all of a sudden. Should we close this discussion?

**Comment by gwhitney** [comment]
Given that there has been no discussion in six months and nobody seems fussed about the speed of `npm run dev`, closing.
